coalib/settings/ConfigurationGathering.py

```diff
@@ -136,9 +136,7 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     cli_sections = parse_cli(arg_list=arg_list, arg_parser=arg_parser)
     check_conflicts(cli_sections)
 
-    if (
-            bool(cli_sections["default"].get("find_config", "False")) and
-            str(cli_sections["default"].get("config")) == ""):
+    if (str(cli_sections["default"].get("config")) == ""):
```

Comment:
Your tests failed, please check what CI told you. It seems that `ConfigurationGatheringTest.test_default_coafile_parsing` failed.